tests/rptest/tests/topic_recovery_test.py

```diff
@@ -7,7 +7,8 @@
 # https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
 
 from rptest.services.cluster import cluster
-from ducktape.mark import ok_to_fail
+from ducktape.mark import ignore
```

Comment:
I think `ignore` is unused import?

Reply:
This was there before I did the ok_to_fail change, so I'm guessing it came back in a revert

Reply:
fixed that